app/src/main/java/org/mozilla/tv/firefox/ext/String.kt

```diff
@@ -85,3 +85,6 @@ val String.isUriYouTubeTV: Boolean
 val String.isUriYouTubeTvVideo: Boolean
     get() = this.isUriYouTubeTV &&
             this.contains("/watch/")
+
+val String.isUriMozSignIn: Boolean
+    get() = this.startsWith("https://accounts.firefox.com/authorization")
```

Comment:
Should this also include the success screen? `accounts.firefox.com/oauth/success`?

Reply:
I don't think they need to really interact with that screen, right?  Might be confusing if there isn't anything to focus.

Reply:
Have you tried it? If there's nothing to focus, maybe nothing happens. I think the cursor can be confusing too: maybe they'll try to click on the "Success" icon.

Then again, once users hit this page, we should redirect them off of it so maybe this is a non-issue.

Reply:
Actually, yeah - we should just pick something and leave it for now because users probably won't see this page at all.

Reply:
I ran through the whole process and didn't notice anything strange about this page, but we can take a closer look before we deploy it to users.  It's a quick and simple tweak.